pages/windows/ftp.md

```diff
@@ -0,0 +1,35 @@
+# ftp
+
+> Interactively transfer files between a local and remote FTP server.
```

Comment:
Windows machines don't have this enabled by default IIRC, so perhaps a note saying `Must be enabled in the Windows features settings` or something might be appropriate?

Reply:
They don't? 🤔 I never enabled it on my PC.